mavros/scripts/mavwp

```diff
@@ -187,6 +207,12 @@ def main():
     load_args.add_argument('-p', '--preserve-home', action='store_true', help="Preserve home location (WP 0, APM only)")
     load_args.add_argument('file', type=argparse.FileType('rb'), help="input file (QGC/MP format)")
 
+    load_partial_args = subarg.add_parser('load_partial', help="partially load waypoints from file")
+    load_partial_args.set_defaults(func=do_load_partial)
+    load_partial_args.add_argument('file', type=argparse.FileType('rb'), help="input file (QGC/MP format)")
+    load_partial_args.add_argument('start_index', type=int, help="waypoint start index")
+    load_partial_args.add_argument('end_index', type=int, help="waypoint end index")
```

Comment:
I think that indexes better set via options, not positional args. Start index should be `required=True`; End - by default use last entry.